src/components/RegisterAccount1.tsx

```diff
@@ -28,48 +21,22 @@ export enum Pages {
 
 export const RegisterAccount1 = (props: RegisterAccount1Props) => {
   const [isLoading, setIsLoading] = useState(false)
-  const [username, setUsername] = useState('')
   const [email, setEmail] = useState('')
   const [page, setPage] = useState(Pages.EMAIL_REGISTRATION)
-  
-  const onSignUpWithGoogle = async (event: React.SyntheticEvent) => {
```

Comment:
I think this is still needed in the Create Account with Google workflow.
![Screenshot 2022-12-19 113020](https://user-images.githubusercontent.com/1864447/208505001-51611c4c-ed69-455d-8c4d-fc513928013d.png)